generate/generate_stack_file_test.go

```diff
@@ -48,13 +48,13 @@ func TestGenerateFile(t *testing.T) {
 			},
```

Comment:
hmm we have generate_root_test and generate_stack_file_test, the naming doesn't seem consistent. Maybe it could be:

* generate_file_root_test.go
* generate_file_stack_test.go

Or:

* generate_root_file_test.go
* generate_stack_file_test.go

Reply:
true!
I changed to use the second option because then we can type less when using file completion to open them. Does it make sense?

Reply:
Totally :100: